### ISSUE TITLE
Livestream download

### ISSUE THREAD

**Issue by robd003** [ihabunek/twitch-dl#66]
Is there a way to download an in-progress live stream?

**Comment by ihabunek** [comment]
Yes but you need to find the video ID which is currently not possible using just twitch-dl. What you can do is go to the streamer's video page, e.g. https://www.twitch.tv/bananasaurus_rex/videos. The current stream should be the first video under "Recent broadcasts". Then copy the URL of the video and feed it to twitch-dl.

**Comment by Kozi801** [comment]
I just discovered this softcware today. Great development. I hope livestream download has improve and or is still posbbile. Can chat be downloaded live as well? or not yet?
